packages/gateway/src/entities/domain.ts

```diff
@@ -0,0 +1,18 @@
+import { Entity, PrimaryColumn, Column, OneToMany } from 'typeorm'
+import { Address } from './address'
+import { Text } from './text'
+
+@Entity()
+export class Domain {
+  @PrimaryColumn()
+  namehash: string
```

Comment:
Namehash is the name of the process, not of the data itself. The [ENS documents](https://docs.ens.domains/terminology#node) clarify that.

Reply:
done